addons/mail/static/src/messaging/component/follower_subtype_list/follower_subtype_list.js

```diff
@@ -22,14 +22,14 @@ class FollowerSubtypeList extends Component {
                 : undefined;
             const followerSubtypes = follower ? follower.subtypes : [];
             return [
```

Comment:
make object instead of array, as it is compareDepth is useless since it uses object key instead of array index